DataMigration/src/test/java/org/sagebionetworks/dian/datamigration/BridgeJavaSdkUtilTests.java

```diff
@@ -0,0 +1,455 @@
+                    .thenReturn(mockDeleteCompletedTestReportCall);
+
+            when(mockUpdateParticipantCall.execute())
+                    .thenReturn(Response.success(new Message()));
+            when(mockParticipantsApi.updateParticipant(anyString(), any()))
+                    .thenReturn(mockUpdateParticipantCall);
+
+            ObjectMapper mapper = new ObjectMapper();
+            // Work-around for BridgeJavaSdk not exposing IdentifierHolder constructor
+            IdentifierHolder identifierHolder = mapper
+                    .readValue("{\"identifier\":\"abc123\"}", IdentifierHolder.class);
+            when(mockSignUpCall.execute())
+                    .thenReturn(Response.success(identifierHolder));
+            when(mockResearcherApi.createParticipant(any()))
+                    .thenReturn(mockSignUpCall);
+
+            when(mockResearcherApi.getParticipantByExternalId(eq("999999"), eq(false)))
+                    .thenThrow(new EntityNotFoundException("Account not found", ""));
+
+            // Work-around for BridgeJavaSdk not exposing user ID
+            StudyParticipant migratedParticipant = mapper
+                    .readValue("{\"id\":\"abc123\"}", StudyParticipant.class);
+            Map<String, String> migratedAttributes = new HashMap<>();
+            migratedAttributes.put(ATTRIBUTE_IS_MIGRATED, ATTRIBUTE_VALUE_TRUE);
+            migratedParticipant.setAttributes(migratedAttributes);
+            when(mockGetExternalIdMigratedCall.execute())
+                    .thenReturn(Response.success(migratedParticipant));
+            when(mockResearcherApi.getParticipantByExternalId(eq("d1a5cbaf-288c-48dd-9d4a-98c90213ac01"), eq(false)))
+                    .thenReturn(mockGetExternalIdMigratedCall);
+
+            // Work-around for BridgeJavaSdk not exposing user ID
+            StudyParticipant notMigratedParticipant = mapper
+                    .readValue("{\"id\":\"abc123\"}", StudyParticipant.class);
+            Map<String, String> notMigratedAttributes = new HashMap<>();
+            notMigratedAttributes.put(ATTRIBUTE_IS_MIGRATED, ATTRIBUTE_VALUE_FALSE);
+            notMigratedParticipant.setAttributes(migratedAttributes);
+            when(mockGetExternalIdNotMigratedCall.execute())
+                    .thenReturn(Response.success(notMigratedParticipant));
+            when(mockResearcherApi.getParticipantByExternalId(eq("999997"), eq(false)))
+                    .thenReturn(mockGetExternalIdNotMigratedCall);
+        }
+    }
+
+    @Test
+    public void test_MigrateUser_AccountNotCreatedYet() throws IOException {
+        HmDataModel.HmUser user = createNewUser();
+        user.externalId = "999999";
+        user.arcId = "999999";
+        HmDataModel.HmUserData data = createFullNewUserData(user.arcId);
+
+        BridgeJavaSdkUtil.migrateUser(user, data);
+        verify(mockSignUpCall).execute();
```

Comment:
This sounds awesome. It does look like I'm targeting JUnit5, maybe I'm missing something with my setup?

`testImplementation("org.junit.jupiter:junit-jupiter:5.8.0")`

Is it because I am using this Mockito version?

`testImplementation("org.mockito:mockito-core:2.23.4")`

Reply:
I know you are not using JUnit 5 because your imports look like "org.junit.*", while JUnit 5 imports look like "org.junit.jupiter.*". Also, JUnit 5 uses `@BeforeEach` rather than `@Before`.  It's fine for this PR. I just wanted to pass along the "FYI."